en/app.ftl

```diff
@@ -150,6 +150,13 @@ whatsnew-feature-offer-countdown-snippet = Get { -brand-name-relay-premium } bef
 whatsnew-feature-offer-countdown-description = Get { -brand-name-relay-premium } before { $end_date } and enjoy unlimited masking at our intro month-to-month price.
 whatsnew-feature-offer-countdown-cta = Upgrade now
 
+whatsnew-feature-firefox-integration-heading = { -brand-name-relay } is now available in { -brand-name-firefox } password manager
+# A preview of the full content of `whatsnew-feature-firefox-integration-description`.
+# When translating, please make sure the resulting string is of roughly similar
```

Comment:
Suggestion to the comment: `Try to keep this string short, not longer than the English version.`

Reply:
Thanks, that's more concise. I applied the same change to the other comments with the same content: 3237718